archive-index.html

```diff
@@ -0,0 +1,897 @@
+                        of things unknown   <br>
+                        but longed for still   <br>
+                        and his tune is heard   <br>
+                        on the distant hill   <br>
+                        for the caged bird   <br>
+                        sings of freedom. <br>
+                        <br>
+                        The free bird thinks of another breeze <br>
+                        and the trade winds soft through the sighing trees <br>
+                        and the fat worms waiting on a dawn bright lawn <br>
+                        and he names the sky his own <br>
+                        <br>
+                        But a caged bird stands on the grave of dreams   <br>
+                        his shadow shouts on a nightmare scream   <br>
+                        his wings are clipped and his feet are tied   <br>
+                        so he opens his throat to sing. <br>
+                        <br>
+                        The caged bird sings   <br>
+                        with a fearful trill   <br>
+                        of things unknown   <br>
+                        but longed for still   <br>
+                        and his tune is heard   <br>
+                        on the distant hill   <br>
+                        for the caged bird   <br>
+                        sings of freedom.<br>
+                      </div>
+                      <div class="modal-footer">
+                        <p class="text-muted">
+                          Maya Angelou, “Caged Bird” from Shaker, Why Don't You Sing? Copyright © 1983 by Maya Angelou.
+                        </p>
+                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                        <button type="button" class="btn btn-primary">Share</button>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+<!-- End Modal -->
+                
+                <hr>
+
+                <!-- Second Poem -->
+                <h2>On the Pulse of Morning</h2>
+                <p  class="text-muted">
+                    by Maya Angelou
+                </p>
+                <p><span class="glyphicon glyphicon-book"></span> Published: 1993</p>
+                <hr>
+                <img class="img-responsive" src="http://placehold.it/900x300" alt="">
+                <hr>
+                <p> 
+                  “The Rock cries out to us today, <br>
+                  You may stand upon me, <br>
+                  But do not hide your face.”
+                </p>
+                
+<!--  Modal -->              
+                <button class="btn btn-primary" data-toggle="modal" data-target="#poem2">Read More <span class="glyphicon glyphicon-chevron-right"></span></button>
+
+                <div class="modal fade" id="poem2" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
```

Comment:
Kira, I really feel like I've learned a lot from reviewing your code on the modals. This is a very cool piece of functionality for our archive, and I love the way the modal fade looks. Excellent job with this.